### PR TITLE
feat(actions): add clear items action

### PR DESCRIPTION
**Pull request by ferdinandsalis**
**What**:
Solves https://github.com/paypal/downshift/issues/186

**Why**:
No way to clear tracked items from children.

**How**:
Added `clearItems` as property method which sets `this.items = []`

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -452,6 +452,7 @@ These are functions you can call to change the state of the downshift component.
 | property                | type                                                             | description                                                                  |
 |-------------------------|------------------------------------------------------------------|------------------------------------------------------------------------------|
 | `clearSelection`        | `function(cb: Function)`                                         | clears the selection                                                         |
+| `clearItems`            | `function()`                                                     | clears the items                                                             |
```
Comment (kentcdodds) [inline README.md]: Could the description be more helpful:

```
Clears downshift's record of all the items. Only really useful if you render your items asynchronously within downshift. See [#186](https://github.com/paypal/downshift/issues/186)
```

File: typings/index.d.ts [hunk 2]
```diff
@@ -1,89 +1,112 @@
-import * as React from 'react';
```
Comment (kentcdodds) [inline typings/index.d.ts]: I guess `kcd-scripts` is working 😅 

We're now applying prettier to `ts` files! :tada:

File: README.md (hunk 1 above)
Comment (ferdinandsalis) [reply README.md]: Yes, it is!

**Comment by tansongyang** [comment]
Yeah, I think the TS definition for `clearItems` makes sense.

**Pull request merged**